share/functions/__fish_complete_suffix.fish

```diff
@@ -8,71 +8,36 @@
 # of $PWD.
 
 function __fish_complete_suffix -d "Complete using files"
+    set -l _flag_prefix ""
+    set -l _flag_comp (commandline -ct)
 
-    # Variable declarations
+    argparse 'prefix=' 'desc=' 'comp=' -- $argv
```

Comment:
Given how few times they're invoked, I think I would prefer long names instead of `desc` and `comp` as we don't really use those anywhere.